### PR TITLE
Translate guides/tutorial/part-1/orientation.md

### LINKED ISSUE #141: Translate `/tutorial/part-1/orientation.md`, Ember 5.1
Please assign yourself to the issue or let a comment at the very moment you start the translation.<br><br>File: `guides/release/tutorial/part-1/orientation.md`<br>From Ember: **N/A**<br>To: **5.1**

### PR DESCRIPTION
**Pull request by BlueCutOfficial**
Closes #141

### CODE REVIEW

File: guides/tutorial/part-1/orientation.md [hunk 1]
```diff
@@ -1,26 +1,24 @@
...
+<img src="/images/tutorial/part-1/orientation/styled-with-tomster@2x.png" alt="L'app Super Rentals à la fin du chapitre" width="1024" height="250">
 
-<img src="/images/tutorial/part-1/orientation/styled-with-tomster@2x.png" alt="The Super Rentals app by the end of the chapter" width="1024" height="250">
+En construisant votre page d'accueil, vous apprendrez à&nbsp;:
 
-While building your landing page, you will learn about:
+- Installer Ember CLI
+- Créer une nouvelle app Ember avec Ember CLI
+- Démarrer et arrêter le serveur de développement
+- Éditer des fichier et les recharger instantanément
```
Comment (MrChocolatine) [inline guides/tutorial/part-1/orientation.md:10]:
```suggestion
- Éditer des fichiers et le rechargement automatiquement
```

File: guides/tutorial/part-1/orientation.md [hunk 2]
```diff
@@ -29,11 +27,11 @@ node: 14.19.1
...
 ```
 
-If a version number is shown, you're ready to go.
+Si un numéro de version s'affiche, vous êtes prêt pour la suite.
 
-## Creating a New Ember App with Ember CLI
+## Créer une nouvelle app Ember avec Ember CLI
 
-We can create a new project using Ember CLI's `new` command. It follows the pattern `ember new <project-name>`. In our case, the project name would be `super-rentals`. We will also include a `--lang en` option. This sets our app's primary language to English and improves the website's [accessibility](../../../accessibility/application-considerations/).
+On crée un nouveau projet en utilisant la commande `new` de Ember CLI. Elle suit la syntaxe `ember new <project-name>`. Dans notre cas, le nom du projet serait `super-rentals`. Nous inclurons aussi une option `--lang en`, qui définit l'anglais comme langue principale de l'app et améliore l'[accessibilité](../../../accessibility/application-considerations/) du site web.
```
Comment (MrChocolatine) [inline guides/tutorial/part-1/orientation.md:34]:
```suggestion
On crée un nouveau projet en utilisant la commande `new` de Ember CLI. Elle suit la syntaxe `ember new <project-name>`. Dans notre cas, le nom du projet serait `super-rentals`. Nous utiliserons aussi l'option `--lang en`, qui définit l'anglais comme langue principale de l'app et améliore l'[accessibilité](../../../accessibility/application-considerations/) du site web.
```

File: guides/tutorial/part-1/orientation.md [hunk 3]
```diff
@@ -161,11 +159,11 @@ super-rentals
 17 directories, 36 files
```
Comment (MrChocolatine) [inline guides/tutorial/part-1/orientation.md:159]:
```suggestion
17 répertoires, 36 fichiers
```

File: guides/tutorial/part-1/orientation.md [hunk 4]
```diff
@@ -161,11 +159,11 @@ super-rentals
...
 ```
 
-We'll learn about the purposes of these files and folders as we go. For now, just know that we'll spend most of our time working within the `app` folder.
+Nous apprendrons le rôle de ces fichiers et dossiers le long du tutoriel. Pour l'instant, sachez seulement que nous passerons la majeure partie du temps à travailler dans le dossier `app`.
 
-## Starting and Stopping the Development Server
+## Démarrer et arrêter le serveur de développement
 
-Ember CLI comes with a lot of different commands for a variety of development tasks, such as the `ember new` command that we saw earlier. It also comes with a _development server_, which we can launch with the `ember server` command:
+Ember CLI fournit beaucoup de commandes différentes adaptées à de nombreuses tâches, comme la commande `ember new` que nous avons vues plus tôt. Ember CLI fournit aussi un serveur de développement ( _development server_), que nous pouvons lancer avec la commande `ember server`:
```
Comment (MrChocolatine) [inline guides/tutorial/part-1/orientation.md:166]: https://cli.emberjs.com/release/advanced-use/cli-commands-reference/#emberserve

```suggestion
Ember CLI fournit beaucoup de commandes différentes adaptées à de nombreuses tâches, comme la commande `ember new` que nous avons vu plus tôt. Ember CLI fournit aussi un serveur de développement ( _development server_), que nous pouvons démarrer avec la commande `ember serve`:
```

File: guides/tutorial/part-1/orientation.md [hunk 5]
```diff
@@ -161,11 +159,11 @@ super-rentals
...
 
 ```shell
 $ ember server
```
Comment (MrChocolatine) [inline guides/tutorial/part-1/orientation.md:169]:
```suggestion
$ ember serve
```

File: guides/tutorial/part-1/orientation.md [hunk 6]
```diff
@@ -174,72 +172,73 @@ building...
...
       </div>
     </div>
     <img src="/images/mascots/zoey.png" role="presentation" alt="">
   </div>
 </div>
 
-You can exit out of the development server at any time by typing `Ctrl + C` into the terminal window where `ember server` is running. That is, typing the "C" key on your keyboard _while_ holding down the "Ctrl" key at the same time. Once it has stopped, you can start it back up again with the same `ember server` command. We recommend having two terminal windows open: one to run the server in background, another to type other Ember CLI commands.
+Vous pouvez stopper le serveur de développement à tout moment en tapant `Ctrl + C` dans le terminal où `ember server` est lancé (c'est-à-dire taper le "C" de votre clavier _pendant_ que vous maintenez la touche "Ctrl" enfoncée). Une fois le serveur arrêté, vous pouvez le démarrer à nouveau avec la même commande `ember server`. Nous vous recommandons d'avoir deux fenêtres de terminal ouvertes&nbsp;: une dans laquelle vous lancez le serveur en tâche de fond, une autre pour taper les autres commandes Ember CLI.
```
Comment (MrChocolatine) [inline guides/tutorial/part-1/orientation.md:191]:
```suggestion
Vous pouvez stopper le serveur de développement à tout moment en tapant `Ctrl + C` dans le terminal où `ember serve` a été lancé (c'est-à-dire taper le "C" de votre clavier _pendant_ que vous maintenez la touche "Ctrl" enfoncée). Une fois le serveur arrêté, vous pouvez le démarrer à nouveau avec la même commande `ember serve`. Nous vous recommandons d'avoir deux fenêtres de terminal ouvertes&nbsp;: une dans laquelle vous lancez le serveur en tâche de fond, une autre pour taper les autres commandes Ember CLI.
```

File: guides/tutorial/part-1/orientation.md [hunk 7]
```diff
@@ -174,72 +172,73 @@ building...
...
 
-## Editing Files and Live Reload
+## Éditer des fichier et les recharger instantanément
```
Comment (MrChocolatine) [inline guides/tutorial/part-1/orientation.md:193]:
```suggestion
## Éditer des fichier et le rechargement automatique
```

File: guides/tutorial/part-1/orientation.md (hunk 1 above)
Comment (BlueCutOfficial) [reply guides/tutorial/part-1/orientation.md:10]:
```suggestion
- Éditer des fichiers et les recharger automatiquement
```

File: guides/tutorial/part-1/orientation.md (hunk 3 above)
Comment (BlueCutOfficial) [reply guides/tutorial/part-1/orientation.md:159]: Not sure we should translate the command output 🤔 is that supposed to depend on the OS language?

File: guides/tutorial/part-1/orientation.md (hunk 4 above)
Comment (BlueCutOfficial) [reply guides/tutorial/part-1/orientation.md:166]:
```suggestion
Ember CLI fournit beaucoup de commandes différentes adaptées à de nombreuses tâches, comme la commande `ember new` que nous avons vue plus tôt. Ember CLI fournit aussi un serveur de développement ( _development server_), que nous pouvons démarrer avec la commande `ember server`:
```

File: guides/tutorial/part-1/orientation.md (hunk 6 above)
Comment (BlueCutOfficial) [reply guides/tutorial/part-1/orientation.md:191]:
```suggestion
Vous pouvez stopper le serveur de développement à tout moment en tapant `Ctrl + C` dans le terminal où `ember server` a été lancé (c'est-à-dire taper le "C" de votre clavier _pendant_ que vous maintenez la touche "Ctrl" enfoncée). Une fois le serveur arrêté, vous pouvez le démarrer à nouveau avec la même commande `ember server`. Nous vous recommandons d'avoir deux fenêtres de terminal ouvertes&nbsp;: une dans laquelle vous lancez le serveur en tâche de fond, une autre pour taper les autres commandes Ember CLI.
```

File: guides/tutorial/part-1/orientation.md (hunk 7 above)
Comment (BlueCutOfficial) [reply guides/tutorial/part-1/orientation.md:193]:
```suggestion
## Éditer des fichier et les recharger automatiquement
```

**Comment by BlueCutOfficial** [comment]
I merge to clean a bit the PR stack, review comments are applied 👍

**Pull request merged**